dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/impl/ProjectServiceImpl.java

```diff
@@ -88,10 +95,11 @@ public class ProjectServiceImpl extends BaseServiceImpl implements ProjectServic
      */
     @Override
     @Transactional
-    public Map<String, Object> createProject(User loginUser, String name, String desc) {
+    public Result createProject(User loginUser, String name, String desc) {
+        Result result = new Result();
 
-        Map<String, Object> result = checkDesc(desc);
-        if (result.get(Constants.STATUS) != Status.SUCCESS) {
+        if (!StringUtils.isEmpty(desc) && desc.length() > 200) {
```

Comment:
Why did you change this logic?

Reply:
instead changing checkDesc(desc);

Reply:
I think it's better to keep the `checkDesc` method, because I found some places are repeated judgment logic.

Reply:
ok, update and keep the checkDesc method. @caishunfeng